packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.js

```diff
@@ -92,33 +88,24 @@ function TranscribedLines ({ lines, scale, task, theme }) {
           const mark = { length, x1, y1, x2, y2 }
           const id = `transcribed-${index}`
           return (
-            <ConsensusLine
-              role='img'
-              aria-describedby={id}
-              aria-label={line.consensusText}
-              key={line.id}
-              onClick={e => createMark(line)}
-              onKeyDown={e => (e.key === 'Enter' && createMark(line))}
-              tabIndex={0}
-            >
-              <TranscriptionLine
-                state='transcribed'
-                mark={mark}
-                scale={scale}
-              />
-              <Tooltip
-                background={theme.global.colors['drawing-red']}
-                className='tooltip'
-                id={id}
-                index={index}
-                label={{
-                  fill: theme.global.colors['neutral-6'],
-                  text: counterpart('TranscribedLines.transcribed')
-                }}
-                x1={x1}
-                y1={y1}
-              />
-            </ConsensusLine>
+            <Tooltip id={id} label={<TooltipLabel fill={fills.transcribed} label={counterpart('TranscribedLines.transcribed')} />}>
+              <ConsensusLine
+                role='img'
+                aria-describedby={id}
+                aria-label={line.consensusText}
+                focusColor={focusColor}
+                key={line.id}
+                onClick={e => createMark(line)}
+                onKeyDown={e => (e.key === 'Enter' && createMark(line))}
```

Comment:
Same as above, I think out of scope, but noting we might want to add space ' ' to also showConsensus(line)